src/Spider.lua

```diff
@@ -655,9 +655,8 @@ function M.spiderSearch(self, dbT, userSearchPat, helpFlg)
 
    local fullT = {}
    for sn, vv in pairs(dbT) do
-      fullT[sn] = sn
       for fn, v in pairs(vv) do
-         fullT[v.fullName] = sn
+         fullT[#fullT+1] = {sn=sn, fullName=v.fullName, fn=fn}
       end
    end
 
```

Comment:
I see that you have changed fullT from using it as a hash table to array.  I understand why you did it but you are going to have to change the name from fullT to fullA.  Otherwise it will confuse me.